app/authentication.go

```diff
@@ -45,10 +45,24 @@ func (a *App) CheckPasswordAndAllCriteria(user *model.User, password string, mfa
 		return err
 	}
 
+	if err := a.CheckUserMfa(user, mfaToken); err != nil {
```

Comment:
Should we first check the password before the MFA? Later when we got Audit v2 rolled out this will be an interesting use case to detect that an attacker has the password, but was stopped by 2FA to warn the user / sys admin in advance. Also when we support different 2FA providers we can avoid an unnecessary API request that way.

Reply:
Agreed. Unless @jwilander had a reason for doing it the other way. I will switch.

Reply:
As long as an attacker doesn't get any feedback about getting the password correct it doesn't matter to me 👍